tests/unit_test/multilingual/multilingual_processor_test.py

```
@@ -618,3 +617,68 @@ def _mock_service_account(*args, **kwargs):
+            "http_action": [{"response": {}}],
+            "email_action": [{}],
+        }
+
+        expected = {
+            "http_action": [{"response": {}}],
+            "email_action": [{}],
+        }
+
+        s_lang = "en"
+        d_lang = "fr"
+        with pytest.raises(Exception):
```

Ruff (0.8.0)

`pytest.raises(Exception)` should be considered evil

(B017)